src/Host/Client/Impl/BrokerServices/WebServer.cs

```diff
@@ -51,62 +65,86 @@ public class WebServer {
                         continue;
                     } catch(ObjectDisposedException) {
                         // Socket got closed
+                        await Log.WriteLineAsync(LogVerbosity.Minimal, MessageCategory.Error, Resources.Error_RemoteWebServerCreationFailed.FormatInvariant(_name));
+                        _console.WriteLine(Resources.Error_RemoteWebServerCreationFailed.FormatInvariant(_name));
                         throw new OperationCanceledException();
                     }
                     break;
                 }
+
+                try {
+                    await Log.WriteLineAsync(LogVerbosity.Minimal, MessageCategory.General, Resources.Info_RemoteWebServerStarted.FormatInvariant(_name, LocalHost, LocalPort));
+                    _console.WriteLine(Resources.Info_RemoteWebServerStarted.FormatInvariant(_name, LocalHost, LocalPort));
+                    _console.WriteLine(Resources.Info_RemoteWebServerDetails.FormatInvariant(Environment.MachineName, LocalHost, LocalPort,_name, RemoteHost, RemotePort));
+                } catch {
```

Comment:
Closing VS while listener just as listener is about to finish finding an available port. My concern was that it might lead to using a invalid console.